### DISCUSSION TITLE
Report event along route?

### DISCUSSION THREAD

**Discussion by mdkcore0** [osmandapp/OsmAnd#11639] [Q&A]
Hi, I'm quite "new" to OsmAnd. Is there something like a "report event" that occured along the route? Like stopped vehicles, traffic jam, car accidents... Can't find it here.

Many thanks you all!

**Comment by thompehw** [comment]
Hello! It's been a while, so as a fellow user I'll offer an answer: no!

OsmAnd is different from Google Maps, Waze, etc. Although it has online features, its focus is to work without an internet connection. Yes, Google can do that too, but since OsmAnd is built around the very free open licenses of the OpenStreetMap project, it doesn't include data sources that are incompatible. 

AFAIK, there aren't any suitably free sources of traffic information that are good enough to be worth having.

Hope that helps! Best wishes.

**Reply by mdkcore0** [reply]
oh, nice, got it!
many thanks, being using OsmAnd as my map application and it's amazing!

**Reply by WKatGit** [reply]
facilmap.org can make a online custom openstreetmap map, and export .gpx routes for osmand. (Maybe someone could make a plugin to share such a custom facil map link through osmand?)
